codingEnvironment.html

```diff
@@ -49,7 +49,7 @@
 
 <div style="height:100%;margin-left:10px;margin-right:10px;">
     <div style=" position:relative; margin-bottom:8px; text-align:center;">
-        <h1>CSE 190 Blockly Playground: Project 3</h1>
+        <h1>Adrian's AWESOME Blockly Playground</h1>
```

Comment:
Not sure I like the AWESOME in all caps - when I open the webpage it looks funny.

We also can't put your name on the title of the website, since this is a project for everyone.